### DISCUSSION TITLE
support for file upload in playground API

### DISCUSSION THREAD

**Discussion by pvanand07** [agno-agi/agno#1466] [Q&A]
Hi everyone,

I am working on building RAG/SQL agents using the Phidata Playground API. I've noticed that the FastAPI endpoint /v1/playground/agent/run includes an additional image request. Could anyone share how to upload a file via this API to use as input for the RAG/SQL tools? Any suggestions on how to implement this, or alternative approaches, would be greatly appreciated.

Thanks in advance!

**phi-playground**
```
POST /v1/playground/agent/run
{
  "message": "string",
  "agent_id": "string",
  "stream": true,
  "monitor": false,
  "session_id": "string",
  "user_id": "string",
  "image": "string"
}
```

**Comment by manthanguptaa** [comment]
Hey @pvanand07, I assume you are making this request using your Python code.
```
image = open('path_to_your_image.jpg', 'rb')
```
and pass it in your request body
